README.md

```diff
@@ -483,6 +483,24 @@ By default all logs from the application aren't shown, if you want to see them,
 
 `./test.sh rest_api --verbose`
 
+### Coverage reports
```

Comment:
Please update the documentation in `docs` directory.

Reply:
done